roles/keycloak_quarkus/templates/keycloak.service.j2

```diff
@@ -8,10 +8,10 @@ Type=simple
 EnvironmentFile=-{{ keycloak_quarkus_sysconf_file }}
 PIDFile={{ keycloak_quarkus_service_pidfile }}
 {% if keycloak_quarkus_start_dev %}
-ExecStart={{ keycloak.home }}/bin/kc.sh start-dev
-{% else %}
-ExecStart={{ keycloak.home }}/bin/kc.sh start --optimized
-{% endif %}
+ExecStart={{ keycloak.home }}/bin/kc.sh start-dev{% if keycloak_quarkus_proxy_headers is defined %} --proxy-headers {{ keycloak_quarkus_proxy_headers }}{% endif -%}{{ '\n' }}
```

Comment:
Not here please, as we try to avoid cmd-line arguments; put an if block inside the config template instead.

Reply:
Do you prefere double if linked (personally i  think it's not easy to read ) : 
```
{% if keycloak_quarkus_start_dev %}
{% if keycloak_quarkus_proxy_headers is defined %}
ExecStart={{ keycloak.home }}/bin/kc.sh start-dev --proxy-headers {{ keycloak_quarkus_proxy_headers }}
{% else -%}
ExecStart={{ keycloak.home }}/bin/kc.sh start-dev
{%- endif %}
{% else -%}
{% if keycloak_quarkus_proxy_headers is defined %}
ExecStart={{ keycloak.home }}/bin/kc.sh start --optimized --proxy-headers {{ keycloak_quarkus_proxy_headers }}
{% else -%}
ExecStart={{ keycloak.home }}/bin/kc.sh start --optimized
{% endif %}
{% endif %}
```

Or a only if (but more complex and elif condition : 
```
{% if keycloak_quarkus_start_dev and keycloak_quarkus_proxy_headers is defined %}
ExecStart={{ keycloak.home }}/bin/kc.sh start-dev --proxy-headers {{ keycloak_quarkus_proxy_headers }}
{% elif keycloak_quarkus_start_dev and keycloak_quarkus_proxy_headers is not defined %}
ExecStart={{ keycloak.home }}/bin/kc.sh start-dev
{% elif not keycloak_quarkus_start_dev and keycloak_quarkus_proxy_headers is defined %}
ExecStart={{ keycloak.home }}/bin/kc.sh start --optimized --proxy-headers {{ keycloak_quarkus_proxy_headers }}
{% else -%}
ExecStart={{ keycloak.home }}/bin/kc.sh start --optimized
{% endif %}
```

Reply:
I've updated my MR with the second choix, but i can use the first one if you prefer

Reply:
Oh sorry, I was unclear; I meant, no arguments added to `kc.sh`; arguments should go instead in the config file template for [keycloak.conf ](https://github.com/ansible-middleware/keycloak/blob/main/roles/keycloak_quarkus/templates/keycloak.conf.j2)

Reply:
I confirm it's much better with this solution.
As i follow "stupidly" the official documentation, i was thinking it's not possible to have it on the keycloak.conf.
i confirm, it's working as expected with configuration on `keycloak.conf`.